tests/end_to_end/workflow/private_attr_wo_callable.py

```diff
@@ -0,0 +1,142 @@
+# Copyright 2020-2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+from openfl.experimental.workflow.interface import FLSpec, Aggregator, Collaborator
+from openfl.experimental.workflow.placement import aggregator, collaborator
+
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger(__name__)
+
+
+class TestFlowPrivateAttributesWoCallable(FLSpec):
+    """
```

Comment:
Can we mention that in this Test the private attributes are initialized directly